google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/RemoveFeedItemAttributeValue.java

```diff
@@ -40,10 +40,9 @@
 import java.util.Map;
 
 /**
- * Removes a the FLIGHT_SALE_PRICE FeedItemAttributeValue in a flights feed. To create a flights
- * feed, run the AddFlightsFeed.java example. This example is specific to feeds of type
- * DYNAMIC_FLIGHT, and specifically removes the FLIGHT_SALE_PRICE attribute, so the
- * FLIGHT_SALE_PRICE attribute must be present on the feed.
+ * Removes a FeedItemAttributeValue in a flights feed. To create a flights
```

Comment:
"Removes a FeedItemAttributeValue of a FeedItem in a flights Feed." This makes it clear that we're operating on an individual feed item, not the feed.